app/components/steps_component_controller.js

```diff
@@ -0,0 +1,67 @@
+// frozen_string_literal: true
+//
+//  Copyright (c) 2012-2023, Schweizer Alpen-Club. This file is part of
+//  hitobito_sac_cas and licensed under the Affero General Public License version 3
+//  or later. See the COPYING file at the top-level directory or at
+//  https://github.com/hitobito/hitobito_sac_cas.
+
+import { Controller } from "controllers"
+
+export default class extends Controller {
+
+  connect() {
+    this.preventKeyboardSubmit()
```

Comment:
Warum kannst du nicht einfach in den Button Click Handlern der Submits this.preventDefault() machen? Machst du ja bei der back Action auch schon